### DISCUSSION TITLE
find AP by knowed station MAC

### DISCUSSION THREAD

**Discussion by fuyonet** [aircrack-ng/aircrack-ng#2589] [General]
hello,

is it possible to find out in what WLAN (what Access point) is a Client, the MAC is knowed?
background:
my Wifi has a filter that my babies can only access allowed sites. But my babies are clever, we have a lot of Wifi's around us and they know the passwords ;)

i know the MAC of the Devices. now I want find out in what wifi they logged in ;)

hope thats possible.

**Comment by ZerBea** [comment]
To monitor the traffic:
`$ tshark -i INTERFACENAME -Y "wlan.ta == 11:22:33:44:55:66"`

To store the traffic to a dump file:
`$ tshark -i INTERFACENAME -f "wlan addr2 50:04:b8:fe:27:40" -w dump.pcapng`

11:22:33:44:55:66 is the MAC of the targeted CLIENT

**Reply by Nonabz** [reply]
Super, thank you. I think in the output i can see the MAC of the other access point. :)

Ich check that out if i see they disconnected ;)

**Comment by ZerBea** [comment]
Exactly that is the purpose of network analyzers like tshark and Wireshark.
The complete filter reference is here:
https://www.wireshark.org/docs/dfref/w/wlan.html
It is possible to filter by what ever you want to monitor the entire traffic or a part of the traffic. The example commented above is a simple example only.

**Comment by tadash10** [comment]
While I understand your concern as a parent, it's important to note that attempting to access or monitor Wi-Fi networks without explicit permission is generally considered unethical and may be illegal. It's crucial to respect the privacy and security of others.

If you want to manage and monitor your children's internet usage, consider discussing and setting up appropriate controls on the devices they use or implementing parental controls at the router level, if available. Many routers have built-in features that allow you to restrict access to certain websites or set up schedules for internet usage.

If you are the administrator of the Wi-Fi network they connect to, you can access the router's administration interface to view connected devices and manage their access. However, attempting to access or monitor other Wi-Fi networks without permission is not recommended and may have legal consequences.

If you are concerned about your children's online activities, open communication and setting clear rules and expectations are often more effective and respectful approaches than attempting to monitor their network usage without their knowledge.
